.dockerignore

```diff
@@ -1 +0,0 @@
-**/.gradle
```

Comment:
wenn nix mehr  drinsteht kann das file auch ganz weg.

Reply:
Entfernt

Reply:
Wie besprochen: File doch wieder zurück, inklusive der einen Zeile

Reply:
Readded file and line

Reply:
ok